### ISSUE TITLE
Burndown chart is not working at all.

### ISSUE THREAD

**Issue by maartenwodan** [ZenHubIO/support#942]
Getting the following error message in the console:

```
chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:7 Possibly unhandled TypeError: Cannot convert undefined or null to object
    at hasOwnProperty (<anonymous>)
    at c (chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:4:29984)
    at Object.t [as has] (chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:5:1040)
    at Object.isKeySelected (chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:40:3087)
    at chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:45:9917
    at w (chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:4:30870)
    at chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:5:25015
    at chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:5:2098
    at Object.t [as map] (chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:5:1040)
    at f (chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:45:9863)i.possiblyUnhandledRejection @ chrome-extension://ogcgkffhplmphkaahpmffcafajaocjbd/js/built/main.js:7
```

Page looks like this:
![image](https://cloud.githubusercontent.com/assets/17009561/19810869/5f533572-9d2f-11e6-90d4-d6cb74d04dd4.png)

Switching milestones only changes the name in the header, but not the dates. The page remains the same.

**Comment by azenMatt** [comment]
Hi @maartenwodan - we have pushed a fix for this in the latest version 2.24.24 - would you be able to manually update and try again?

**Comment by maartenwodan** [comment]
Yes, updated and it works again! Thanks! Don't know what your policy is on opening and closing issues, but closed it for you ;).
